### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by tushartiwari7**
User can add a comment to some tweet.

### CODE REVIEW

File: src/Components/AddCommentBox/AddCommentBox.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    (state: any) => state.tweets.commentsLoading
+  );
+  const onReset = () => {
+    form.resetFields();
+  };
+
+  const onFinish = async () => {
+    try {
+      const values = await form.validateFields();
+      console.log(values);
```
Comment (mariyasada) [inline src/Components/AddCommentBox/AddCommentBox.tsx:35]: remove console.log

**Review by mariyasada — APPROVED** [review]
code looks clean.remove console log

File: src/Pages/Tweet/Tweet.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+const CommentList = ({ comments }: commentList) => {
+  const commentsLoading = useSelector(
+    (state: any) => state.tweets.commentsLoading
+  );
+  return (
+    <List
+      dataSource={comments}
+      loading={commentsLoading}
+      renderItem={(props: any) => {
+        console.log(props);
```
Comment (KanekarSnehal) [inline src/Pages/Tweet/Tweet.tsx:25]: remove console.log

**Review by KanekarSnehal — APPROVED** [review]
Code looks good to merge, no issues found

**Pull request merged**